code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm

```diff
@@ -93,17 +93,13 @@
 	color = "#BF0000"
 	overdose = REAGENTS_OVERDOSE
 	scannable = 1
-	metabolism = REM * 1.5//Get to overdose state a bit faster
+	metabolism = REM * 0.5
 	taste_description = "bitterness"
 	taste_mult = 3
-	breathe_met = REM * 1.5 * 0.5
-	breathe_mul = 0.5
 
 /datum/reagent/bicaridine/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
 	M.heal_organ_damage(5 * removed, 0)
-	if(M.getBruteLoss() && prob(7))
-		to_chat(M, span("warning", "Your muscles spasm and you find yourself unable to stand!"))
-		M.Weaken(3)
+	M.drowsyness = max(M.drowsyness, 5)
```

Comment:
Uh. If the code doesn't add drowsiness, then why is it capping it? This line should be removed or modified to like, `M.drowsyness = max(M.drowsyness + 1, 5)` Also, it's technically bad to do it this way, since another chemical may have another cap and they'll start overriding themselves. The proper way to do this would be:
```dm
if (M.drowsyness < 5)
	M.drowsyness++
```

Reply:
Actually, nvm, early morning reviews are good. I see what it does now. It ensures a minimal level of drowsyness each tick.

Carry on.